llvm/test/CodeGen/ARM/apple-version-min.ll

```diff
@@ -0,0 +1,9 @@
+; Test emitting version_min directives.
+
+; RUN: llc %s -filetype=asm -o - --mtriple arm64-apple-tvos9.0.0 | FileCheck %s --check-prefix=TVOS
```

Comment:
This is not the right directory for AArch64 tests (which apple calls `arm64` in its triples), these tests will only be run if the ARM backend is enabled, whereas I think this line only wants to be run if the AArch64 backend is enabled.

Reply:
Right, this test has one arm64 command and two thumb commands. 

I just now pushed #95106 to add aarch64-registered-target to this test. I did that because the different options are thematically related and splitting into two tests seemed not in the best interests of test coherency.

Reply:
That fix makes sense to me. Thanks!